wgpu-core/src/command/render.rs

```diff
@@ -979,6 +989,23 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                                 }
                             }
                         }
+
+                        // Clear push constant ranges
+                        let non_overlapping = super::bind::compute_nonoverlapping_ranges(
+                            &pipeline_layout.push_constant_ranges,
+                        );
+                        for range in non_overlapping {
+                            let clear_bytes =
+                                vec![0_u32; (range.range.end - range.range.start) as usize / 4];
```

Comment:
I think the way we'd set this up in the near future (or now?) is: having a fixed-size zero vector somewhere, say in `Device`, say of 256 bytes. Then this code would have a loop that re-uses the same vector and calls `push_graphics_constants` as many times as needed to fill in zeroes.